app/facades/search_facade.rb

```diff
@@ -0,0 +1,6 @@
+class SearchFacade
+  def self.search_gifs(query)
+    results = GifService.search_gifs_api(query) # step 1. runs the query
+    results[:data].map { |data| Gif.new(data) } # step 2. transforms into object
```

Comment:
Good use of single line statement. Facade is light and seems like it's doing what it's supposed to.